CopilotKit/packages/react-textarea/src/components/hovering-toolbar/hovering-toolbar.tsx

```diff
@@ -51,57 +53,82 @@ export const HoveringToolbar = (props: HoveringToolbarProps) => {
 
     const domRange = domSelection.getRangeAt(0);
     const rect = domRange.getBoundingClientRect();
-
+    
     // We use window = (0,0,0,0) as a signal that the selection is not in the original copilot-textarea,
     // but inside the hovering window.
     //
     // in such case, we simply do nothing.
     if (rect.top === 0 && rect.left === 0 && rect.width === 0 && rect.height === 0) {
       return;
     }
-    const minGapFromEdge = 60;
-    const verticalOffsetFromCorner = 35;
-    const horizontalOffsetFromCorner = 15;
-    let top = rect.top + window.scrollY - el.offsetHeight + verticalOffsetFromCorner;
```

Comment:
el.offsetHeight here doesn't make any sense because it renders as an empty div, so its height will be equal to 1em by default (just block paddings). Also, it's weird to position it 35px from the top side of the selection. It must use the bottom if we're trying to position the modal below the selection.